cinema_service/settings.py

```diff
@@ -24,7 +25,7 @@
     "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
```

Comment:
The `SECRET_KEY` is hardcoded and exposed in the settings file. This is a security risk, especially if the code is shared publicly. Consider using environment variables to securely manage the secret key.